API/Controllers/TripController.cs

```diff
@@ -33,5 +34,21 @@ public async Task<ActionResult> CreateTripAsync([FromBody] CreateTripDTO createT
 
             return Ok();
         }
+
+        [HttpPost("add-offers")]
+        [AuthorizeByRole(IdentityRoleNames.Logist)]
+        public async Task<ActionResult> AddOfferToTrip([FromBody] OffersForTripDTO offersForTrip)
```

Comment:
Add "Async"

Reply:
The same. in the plural - AddOffersToTripAsync